markov.py

```diff
@@ -60,6 +61,16 @@ def remove_messages(message):
     bot.reply_to(message, 'u r not an admin 🤔')
 
 
+@bot.message_handler(commands=['version'])
+def get_repo_version(message):
+	hash_len = 7
```

Comment:
could you change from tabs to spaces please? 4 spaces :)

Reply:
Done :)